internal/storage/memory/memory_test.go

```diff
@@ -0,0 +1,153 @@
+package memory
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gofiber/fiber/v2/utils"
+)
+
+var testStore = New()
+
+func Test_Memory_Set(t *testing.T) {
```

Comment:
We should make these names different. e.g `Test_Storage_Memory`

Reply:
done